packages/runtime-node/src/node-environments-manager.ts

```diff
@@ -424,28 +431,24 @@ export class NodeEnvironmentsManager {
         };
     }
 
-    private async getConfig(configName: string | undefined) {
+    private async getConfig(configName: string | undefined, envName: string, basePath: string) {
         const config: TopLevelConfig = [];
         const { configurations } = this.options;
         if (configurations && configName) {
-            const configDefinition = configurations.get(configName);
-            if (!configDefinition) {
+            const foundConfig = await findAllConfigs({
```

Comment:
`getConfig` now re-searches for configs on every call. is that really the behavior we want?

Reply:
this is exactly what happened before

```
 for (const definition of configDefinition) {
                try {
                    if (Array.isArray(definition)) {
                        config.push(...definition);
                    } else {
                        config.push(...((await import(definition.filePath)) as { default: TopLevelConfig }).default);
                    }
                } catch (e) {
                    console.error(e);
                }
            }
```